src/Adapter/Module/Tab/ModuleTabRegister.php

```diff
@@ -86,18 +86,22 @@ public function __construct(TabRepository $tabRepository, LangRepository $langRe
         $this->filesystem = $filesystem;
         $this->languages = $languages;
     }
-    
+
     /**
      * Fetch module-defined tabs and find undeclared ModuleAdminControllers.
      *
      * This is done automatically as part of the module installation.
      * @param Module $module
-     * 
+     *
      */
     public function registerTabs(Module $module)
     {
+        if (!$module->getInstance()) {
```

Comment:
This works obviously, but FYI there is a specific function returning a boolean to check if we can get the instance:

```php
PrestaShop\PrestaShop\Adapter\Module\Module::hasValidInstance()
```